src/desdemona/launcher/aeron_media_driver.clj

```diff
@@ -1,11 +1,38 @@
 (ns desdemona.launcher.aeron-media-driver
   (:gen-class)
-  (:require [clojure.core.async :refer [chan <!!]])
+  (:require [clojure.core.async :refer [chan <!!]]
+            [clojure.tools.cli :refer [parse-opts]])
   (:import [uk.co.real_logic.aeron Aeron$Context]
            [uk.co.real_logic.aeron.driver MediaDriver MediaDriver$Context ThreadingMode]))
 
+(def cli-options
+  [["-t" "--threading-mode THREADING-MODE" "Aeron media driver threading mode: shared or dedicated or shared-network"
+    :default "shared"
+    :validate [#{"shared" "dedicated" "shared-network"} "Must be shared, dedicated, or shared-network"]]
+   ["-d" "--delete-dirs" "Delete the media drivers directory on startup" :default false]
+   ["-h" "--help"]])
```

Comment:
I'm not sure we need the threading-mode if we just target docker-compose either.